omni_agent/vision/vision_agent.py

```
@@ -0,0 +1,182 @@
+                {"type":"text","label":"Hero Heading","bbox":[0.08,0.13,0.92,0.27],"issues":[],"confidence":0.97},
+                {"type":"button","label":"CTA Button","bbox":[0.34,0.40,0.66,0.50],"issues":[],"confidence":0.98},
+                {"type":"input","label":"Email Field","bbox":[0.24,0.53,0.76,0.61],"issues":[],"confidence":0.96},
+                {"type":"card","label":"Feature A","bbox":[0.02,0.66,0.32,0.84],"issues":[],"confidence":0.87},
+                {"type":"card","label":"Feature B","bbox":[0.35,0.66,0.65,0.84],"issues":[],"confidence":0.87},
+                {"type":"card","label":"Feature C","bbox":[0.68,0.66,0.98,0.84],"issues":[],"confidence":0.87},
+            ],
+            issues=[],
+            insights="All violations resolved. 3-column grid detected. UI ready!",
+            score=91,
+        ),
+    ]
```

Ruff (0.15.2)

[warning] Mutable default value for class attribute

(RUF012)